listener.go

```
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"log"
+
+	"github.com/JSainsburyPLC/ingestion-sdk/bus"
+	"github.com/JSainsburyPLC/ingestion-validation-service/configuration"
+)
+
+func listen(c *configuration.Configuration) bus.Delivery {
+	session, err := bus.NewSession(c.RabbitMQ.Dial)
+	if err != nil {
+		log.Fatal(err)
+	}
+
```

log.Fatal() or we could return an error?